src/panels/lovelace/tile-features/hui-cover-position-tile-feature.ts

```diff
@@ -0,0 +1,111 @@
+import { HassEntity } from "home-assistant-js-websocket";
+import { css, html, LitElement, nothing } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import { computeDomain } from "../../../common/entity/compute_domain";
+import { supportsFeature } from "../../../common/entity/supports-feature";
+import { CoverEntityFeature } from "../../../data/cover";
+import { HomeAssistant } from "../../../types";
+import { LovelaceTileFeature } from "../types";
+import { CoverPositionTileFeatureConfig } from "./types";
+import { stateActive } from "../../../common/entity/state_active";
+import { computeAttributeNameDisplay } from "../../../common/entity/compute_attribute_display";
+import { UNAVAILABLE } from "../../../data/entity";
+
+export const supportsCoverPositionTileFeature = (stateObj: HassEntity) => {
+  const domain = computeDomain(stateObj.entity_id);
+  return (
+    domain === "cover" &&
+    supportsFeature(stateObj, CoverEntityFeature.SET_POSITION)
+  );
+};
+
+@customElement("hui-cover-position-tile-feature")
+class HuiCoverPositionTileFeature
+  extends LitElement
+  implements LovelaceTileFeature
+{
+  @property({ attribute: false }) public hass?: HomeAssistant;
+
+  @property({ attribute: false }) public stateObj?: HassEntity;
+
+  @state() private _config?: CoverPositionTileFeatureConfig;
+
+  static getStubConfig(): CoverPositionTileFeatureConfig {
+    return {
+      type: "cover-position",
+    };
+  }
+
+  public setConfig(config: CoverPositionTileFeatureConfig): void {
+    if (!config) {
+      throw new Error("Invalid configuration");
+    }
+    this._config = config;
+  }
+
+  protected render() {
+    if (
+      !this._config ||
+      !this.hass ||
+      !this.stateObj ||
+      !supportsCoverPositionTileFeature(this.stateObj)
+    ) {
+      return nothing;
+    }
+
+    const percentage = stateActive(this.stateObj)
+      ? this.stateObj.attributes.current_position ?? 0
+      : 0;
+
+    const value = Math.max(Math.round(percentage), 0);
+
+    return html` <div class="container">
+      <ha-control-slider
+        .value=${value}
+        min="0"
+        max="100"
+        .step="1"
```

Comment:
The default value of step in ha-control-slider.ts is 1. Why should we be explicit here?

Reply:
I was wondering if the cover has the option to set the step to a different value, but I didn't find this. 
So if there is no such option we can remove this line.

Reply:
I assume you are using the Demo Integration in the development environment to test this feature.
The demo covers provided by the Demo integration cannot be set to single percent values.
[Here](https://github.com/home-assistant/core/blob/95109072b523d47525421ee5f136a5ff23dd7e99/homeassistant/components/demo/cover.py#L192) you can see that the target position (same goes for tilt) is rounded in 10 percent steps.

If you go from 20% to 21% percent the demo cover will close/open fully as a result of rounding the value (this behavior could be considered a bug).

Reply:
@TillFleisch I had a hard time trying to use my HA instance with the frontend during the development, so yes, I set up a new instance in dev containers and used the demo integration.

But the idea behind the step was to allow the user to customize the step size, but let's remove that for now.